lte/gateway/c/oai/tasks/nas/emm/Authentication.c

```diff
@@ -1486,6 +1489,23 @@ static void nas_itti_auth_info_req(
   auth_info_req->visited_plmn  = *visited_plmnP;
   auth_info_req->nb_of_vectors = num_vectorsP;
 
+  /*
+   * Check if we have UE 5G-NR
+   * connection supported in Attach request message.
+   * This is done by checking either en_dc flag in ms network capability or
+   * by checking  dcnr flag in ue network capability.
+   */
+
+  if (dcnr) {
+    auth_info_req->supportedfeatures.nr_as_secondary_rat = 1;
+  } else {
+    auth_info_req->supportedfeatures.nr_as_secondary_rat = 0;
+    OAILOG_DEBUG(
+        TASK_MME_APP,
+        "UE is connected on LTE, Dual registration with 5G-NR is disabled for "
```

Comment:
this is missing the ue_id param to be logged

Reply:
I had used this log as a debug msg which is now removed.